Remove jdbc and jms driver jar files from docker image (#10180)
frankframework/frankframework: Docker.md

### Drivers

Please provide the necessary JDBC and JMS drivers for your database and JMS provider. You can mount the driver JARs to the `/opt/frank/drivers` directory in the container.

> This allows you to use your own drivers without having to rebuild the image. By default, we only provide the H2 JDBC driver in the image to have a sensible default available. 

Reviewer: ```suggestion
This allows you to use your own drivers without having to rebuild the image. By default, we only provide the H2 JDBC driver in the image for local development. 
```
